ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/LedgerDao.scala

```diff
@@ -167,10 +170,13 @@ trait LedgerReadDao extends ReportsHealth {
 
 trait LedgerWriteDao extends ReportsHealth {
 
+  /** The maximum number of concurrent connections */
```

Comment:
Same here, adding this to the name makes more sense.

Reply:
Done.